tests/robot-tests/tests/main.robot

```diff
@@ -0,0 +1,38 @@
+*** Settings ***
+Documentation  DFE-115 Setup automated testing framework
+
+Resource    keywords_and_variables.robot
+Resource    ../pages/HomePage.robot
+Resource    ../pages/ThemesPage.robot
+
+Suite Setup       user opens the browser
+Suite Teardown    user closes the browser
+
+*** Test Cases ***
+Validate Schools page displays links to publications
+    [Tags]  HappyPath
+    user goes to url  ${url}
+    user clicks link  ${HomePage_ThemesLink}
+    user clicks link  ${ThemesPage_SchoolsLink}
+
+    css should match x times  css:[data-testid="data-list-element"]  6
+    element should contain  css:[data-testid="absence-and-exclusions"]  Absence and exclusions
```

Comment:
I know these haven't defined yet, but it seems like you would need to dynamically generate test IDs for these links. Would it be better to select on link text?